IronGram/src/main/java/com/theironyard/services/ExecutionerOfPhotos.java

```diff
@@ -0,0 +1,28 @@
+package com.theironyard.services;
+
+import com.theironyard.entities.Photo;
+
+import java.util.List;
+
+/**
+ * Created by EddyJ on 7/31/16.
+ */
+public class ExecutionerOfPhotos implements Runnable {
+    private PhotoRepository photoRepository;
+    private Photo photo;
+
+    public ExecutionerOfPhotos(PhotoRepository photoRepository, Photo photo) {
+        this.photoRepository = photoRepository;
+        this.photo = photo;
+    }
+
+    @Override
+    public void run() {
```

Comment:
One thing to be clear on is if anything interrupts this thread for any reason the system will delete the photo immediately.  Not a huge deal for this application but be very careful in that in another situation this may be detrimental for timing.